examples/animation/main.rs

```diff
@@ -243,23 +380,46 @@ fn initialise_animation(world: &mut World) -> Handle<Animation<Transform>> {
             [-1., 0., 0., 0.].into(),
         ],
     };
-    let translation_animation_handle =
+    let translation_sampler_handle =
         loader.load_from_data(translation_sampler, (), &world.read_resource());
-    //let scale_animation_handle = loader.load_from_data(scale_sampler, &world.read_resource());
-    let rotation_animation_handle =
+    let scale_sampler_handle = loader.load_from_data(scale_sampler, (), &world.read_resource());
+    let rotation_sampler_handle =
         loader.load_from_data(rotation_sampler, (), &world.read_resource());
-    let animation = Animation {
-        nodes: vec![
-            (
-                0,
-                TransformChannel::Translation,
-                translation_animation_handle,
-            ),
-            //(0, scale_animation_handle),
-            (0, TransformChannel::Rotation, rotation_animation_handle),
-        ],
-    };
-    loader.load_from_data(animation, (), &world.read_resource())
+    let animation_storage = world.read_resource();
+    let mut set = AnimationSet::<AnimationId, Transform>::new();
+    set.insert(
```

Comment:
Can you make a function please?

Reply:
Not sure it made it better, but there you go.